workflow/scores/precision_recall.py

```diff
@@ -226,12 +226,11 @@ def __init__(self, name='precision', precision=2, conf_threshold=0.5):
         self.conf_threshold = conf_threshold
 
     def __call__(self, y_true, y_pred, conf_threshold=None):
-        if conf_threshold is None:
-            conf_threshold = self.conf_threshold
```

Comment:
this should not be removed (you also did not do it for the other occurrences) 

(it's maybe not the nicest code, but when removing it should be solved in another way)